docs/reference/cli.md

```diff
@@ -361,7 +361,7 @@ Options:
   --help  Show this message and exit.
 ```
 
-**Caution**: contact your SQLMesh administrator before running this command.
+**Caution**: this command affects all SQLMesh users. Contact your SQLMesh administrator before running.
```

Comment:
Should we copy this over to `migrate` as well?

Reply:
Good call!